spec/models/survey_spec.rb

```diff
@@ -125,12 +125,12 @@
       s2.questions << q3
     end
 
-    it "includes title, sections, and questions" do
+    xit "includes title, sections, and questions" do
```

Comment:
Should this be back to `it`?

Reply:
Revision -- should we remove this whole section since we removed the `as_json` method from this model in removing `rabl`?

Reply:
I initially had these removed but then wasn't totally sure if we wanted to keep them JIC we added this code to our repo and needed to ensure the overwritten `as_json` worked as expected. Probably a YAGNI